pipeline/metadata/schema.py

```diff
@@ -246,13 +260,32 @@ def flatten_for_bigquery_satellite(row: SatelliteRow) -> Dict[str, Any]:
   }
 
   for received_answer in row.received:
-    answer = {
+    http_response = received_answer.http_response or HttpsResponse()
+    https_response = received_answer.https_response or HttpsResponse()
+
+    answer: Dict[str, Any] = {
         'ip': received_answer.ip,
-        'asnum': received_answer.ip_metadata.asn,
-        'asname': received_answer.ip_metadata.as_name,
         'http': received_answer.http,
         'cert': received_answer.cert,
         'matches_control': received_answer.matches_control,
+        # Ip Metadata
+        'asnum': received_answer.ip_metadata.asn,
+        'asname': received_answer.ip_metadata.as_name,
+        # HTTP
+        'http_response_is_known_blockpage': http_response.is_known_blockpage,
+        'http_response_page_signature': http_response.page_signature,
+        'http_response_status': http_response.status,
+        'http_response_body': http_response.body,
+        'http_response_headers': http_response.headers,
```

Comment:
Minor style suggestion: let's order the fields in the order we observe or derive them:
```suggestion
        'http_response_status': http_response.status,
        'http_response_headers': http_response.headers,
        'http_response_body': http_response.body,
        'http_response_page_signature': http_response.page_signature,
        'http_response_is_known_blockpage': http_response.is_known_blockpage,
```

Same for HTTPS